docs/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 ## [Unreleased]
 
 ## [0.11.0] - 2019-06
+* Fixes and renames regression from ([#1442]) added back `position_from_world` as `screen_to_world`. Also added
```

Comment:
This needs to go abovs 0.11. Please also add a subsection above.

Reply:
There alread is an `## [Unreleased]` section. Just move that line above :)

Reply:
Fixed.